service/project.go

```diff
@@ -343,54 +388,6 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var hook *model.GithubHook
-	hook, err = model.FindGithubHook(responseRef.Owner, responseRef.Repo)
-	if err != nil {
-		uis.LoggedError(w, r, http.StatusInternalServerError, err)
-		return
-	}
-	origGithubWebhookEnabled := hook != nil
-	currentGithubWebhookEnabled := origGithubWebhookEnabled
-	if responseRef.SetupGithubHook && hook == nil {
-		var hookID int
-		hookID, err = uis.setupGithubHook(projectRef)
```

Comment:
This caused a bug where if the owner/repo were changed on this save we would try to create a hook for the _old_ names (from the existing projectRef and not from the request). You would have to try a second time to get it to work.